data/default_release_sources.json

```diff
@@ -0,0 +1,218 @@
+[
```

Comment:
I haven't followed the original change, but I just wonder why is this object separated from default_release_images.json? Is it just for simplicity, or is there some other consideration? Also, do we handle updating these values with the upgrade script (i.e. when a new OCP version is released, etc)?